app/views/admin/creator_settings/_form.html.erb

```diff
@@ -8,20 +8,24 @@
 </div>
 
 <div class="crayons-field mt-6 align-left">
-  <%= label_tag :logo_svg, class: "crayons-field__label" do %>
+  <div class="crayons-field__label relative">
     Logo
     <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
     <p id="logo-subtitle" class="crayons-field__description">Ideally SVG, but PNG will work, too. Min size: 300x300px.</p>
-  <% end %>
-  <span class="block border-none">
-    <%= file_field_tag :logo_svg, class: "crayons-btn crayons-btn--secondary  w-100 ", role: "button", required: true %>
-  </span>
-
-  <% if ::Settings::General.logo_svg.present? %>
-    <div class="site-logo">
-      <%= ::Settings::General.logo_svg.html_safe %>
+  </div>
+  <div class="flex gap-2">
+    <div class="flex items-center">
+      <%= file_field_tag :logo_svg, class: "screen-reader-only", required: true, accept: "image/*", data: { "max-file-size-mb": "25", action: "change->creator-settings#previewLogo" } %>
+      <label class="cursor-pointer crayons-btn crayons-btn--s crayons-btn--secondary" for="logo_svg">
+        Upload Logo
+      </label>
     </div>
-  <% end %>
+    <div class="site-logo" aria-hidden="true" data-creator-settings-target="previewLogo">
```

Comment:
I moved this around, because it looks like if there is a logo, it will load it. I took the `<div />` out of the `if` so I can use the same div for dynamic previews.

Reply:
I can't decide if this should be `aria-hidden` or if the images should just have alt text confirming e.g. the filename or "currently selected logo" or smth.

I'm just thinking about it from the POV of not all screen reader users being visually impaired and wondering what this block is

Reply:
Yeah, to me it seem like a visual enhancement only since the file selection should be enough, but im flip flopping still.

Reply:
I guess an empty `alt` tag on the image would do the same job too and avoid using ARIA?

I think yeah there's maybe not much to describe since it's just the current file input value really.

Reply:
I flip-flopped on this again. Now there is alt text that says `preview of logo selected` on the image and I updated the E2E test to check for it. I think it makes sense to let them know the image preview is there, but let me know what you think.

Reply:
I think that's grand. Could potentially be helpful for screen reader users who can perceive the image well